templates/repo/view_file.tmpl

```diff
@@ -38,7 +38,11 @@
 	<div class="ui attached table segment">
 		<div class="file-view {{if .IsRenderedHTML}}markdown{{else if .IsTextFile}}code-view{{end}} has-emoji">
```

Comment:
Please add here `{{.IsMarkup}}markdown{{else}}plain-text{{end}}` in place of `markdown` and in `_repository.less` file after line 271 add style:
```css
			.plain-text {
				padding: 1em 2em 1em 2em;
			}
```

I think this way looks much better:
![attels](https://user-images.githubusercontent.com/165205/31636204-2d701656-b2d2-11e7-9778-5c3f5447d090.png)

Reply:
This looks actually like this:


![screenshot2](https://user-images.githubusercontent.com/5757182/31636749-075028ae-b2cc-11e7-8116-ec99a6d2d0cb.png)

Is this still okay? I've left the `<pre>` stuff in place.

Reply:
Should be fine in all, try cleaning cache

Reply:
> also did you do make generate-stylesheets?

Yes